apps/src/code-studio/pd/workshop_enrollment/enroll_form.jsx

```diff
@@ -634,6 +655,71 @@ export default class EnrollForm extends React.Component {
           </div>
         )}
 
+        {this.props.workshop_course === CSP &&
+          this.props.workshop_subject ===
+            SubjectNames.SUBJECT_CSP_FOR_RETURNING_TEACHERS && (
+            <div>
+              <FieldGroup
+                id="years_teaching"
+                label="Years Teaching (overall)"
+                type="text"
+                required={true}
+                onChange={this.handleChange}
+                validationState={
+                  this.state.errors.hasOwnProperty('years_teaching')
+                    ? VALIDATION_STATE_ERROR
+                    : null
+                }
+                errorMessage={this.state.errors.years_teaching}
+              />
+              <FieldGroup
```

Comment:
Note there's no validation here to confirm whether the value is numeric.

Reply:
Interesting - that would be nice to add, I will take a look today.

Reply:
Wait, is this by design?  `FieldGroup` has number validation built-in if you change the type to `"number"`:

https://github.com/code-dot-org/code-dot-org/blob/cd6f7b8c75edb819f33c8c0d12a9e85ab3997fcd/apps/src/code-studio/pd/form_components/FieldGroup.jsx#L22-L27